frb_codegen/src/source_graph.rs

```diff
@@ -183,10 +183,11 @@ impl Debug for Module {
 }
 
 /// Get a struct or enum ident, possibly remapped by a mirror marker
-fn get_ident(ident: &Ident, attrs: &[Attribute]) -> (Ident, bool) {
+fn get_ident(attrs: &[Attribute]) -> Vec<Ident> {
     markers::extract_mirror_marker(attrs)
-        .and_then(|path| path.get_ident().map(|ident| (ident.clone(), true)))
-        .unwrap_or_else(|| (ident.clone(), false))
+        .into_iter()
+        .filter_map(|path| path.get_ident().map(|ident| ident.clone()))
```

Comment:
uber-nit: Do we need that clone?

Reply:
Yes, `clone` is needed here, the `ident` returned must be owned, as it was in previous versions.

Reply:
Oh yes, I did not check types when viewing it in github